src/Network/Minio/Data.hs

```diff
@@ -184,21 +184,31 @@ type Object = Text
 -- TODO: This could be a Sum Type with all defined regions for AWS.
 type Region = Text
 
--- | A type alias to represent an Entity-Tag returned by S3-compatible
--- APIs.
+-- | A type alias to represent an Entity-Tag returned by S3-compatible APIs.
 type ETag = Text
 
 -- |
 -- Data type represents various options specified for PutObject call.
 -- To specify PutObject options use the poo* accessors.
 data PutObjectOptions = PutObjectOptions {
+  -- | A standard MIME type describing the format of the contents.
     pooContentType        :: Maybe Text
+  -- | Specifies what content encodings have been applied to the object and thus
+  -- what decoding mechanisms must be applied to obtain the media-type
+  -- referenced by the Content-Type header field.
   , pooContentEncoding    :: Maybe Text
+  -- | Specifies presentational information for the object.
   , pooContentDisposition :: Maybe Text
+  -- | Can be used to specify caching behavior along the request/reply chain.
   , pooCacheControl       :: Maybe Text
+  -- | Entity header is used to describe the language(s) intended
+  -- for the audience.
   , pooContentLanguage    :: Maybe Text
+  -- | If you don't specify, Standard is the default storage class.
```

Comment:
We could say "Possible values are Standard and ReducedRedundancy. The storage class is Standard by default (i.e when Nothing is passed)."